Firestore/core/src/firebase/firestore/util/suppress_warnings.h

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
This filename seems over-specific. You might consider naming this file just `warnings.h`, which would give us the opportunity to add macros in here that mark things deprecated. Or maybe just make this `macros.h`?

No strong feelings. Feel free to leave as-is.